### PR TITLE
Adds client authorization filter

### PR DESCRIPTION
**Pull request by ed-**
See https://repose.atlassian.net/wiki/display/REPOSE/Client+Authorization+filter

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -3,19 +3,15 @@ driver_plugin: vagrant
 driver_config:
   require_chef_omnibus: true
 
+provisioner:
+  name: chef_zero
+  require_chef_omnibus: <%= ENV['BOOTSTRAP_VERSION'] || '11.16.4' %>
+
```
Comment (filler) [inline .kitchen.yml]: This is site-specific.  We shouldn't drop such constraints in a community cookbook.

File: .kitchen.yml [hunk 2]
```diff
@@ -3,19 +3,15 @@ driver_plugin: vagrant
...
     box: opscode-ubuntu-14.04
     box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-14.04_chef-provisionerless.box
-- name: ubuntu-12.04
-  driver_config:
-    box: opscode-ubuntu-12.04
-    box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-12.04_chef-provisionerless.box
-- name: centos-6.5
-  driver_config:
-    box: opscode-centos-6.5
-    box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_centos-6.5_chef-provisionerless.box
```
Comment (filler) [inline .kitchen.yml]: We support CentOS and Ubuntu `12.04`.  Lets keep this going in the community cookbook.

**Comment by ed-** [comment]
oh i have to increment metadata.rb

File: .kitchen.yml (hunk 1 above)
Comment (ed-) [inline .kitchen.yml]: /me shakes fist at eric

File: metadata.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 license          "All rights reserved"
 description      "Installs/Configures repose"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.10.0"
+version          "2.10.1"
```
Comment (filler) [inline metadata.rb]: Seems like a nit, but I would make this a minor release (`2.11.0`) versus a bug release (`2.10.1`).

**Comment by filler** [comment]
Ping @b0d0nne11

**Comment by filler** [comment]
It looks like I lost write to this repo.  :cry: 

Ping @dimtruck

**Pull request merged**